.github/workflows/release_pr.yml

```diff
@@ -0,0 +1,60 @@
+name: Create PR for a release
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: 'Release from which branch?'
+        required: true
+        default: 'main'
+      git_user_name:
+        description: 'Git user name'
+        required: true
+        default: 'devops'
+      git_user_email:
+        description: 'Git user name'
+        required: true
+        default: 'devops@test.com'
+jobs:
+  create_pr_for_next_release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Update git
+      run: |
+        sudo add-apt-repository -y ppa:git-core/ppa
+        sudo apt-get update
+        sudo apt-get install git -y
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.inputs.base_branch }}
+        token: ${{ github.token }} # This will need to change to a token belonging to a service account.
```

Comment:
github.token is an internal token that is tied to the Github Workflows app installation on the current repository. It has limited scope but it's good enough to do things like checking out code. https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

In this particular case, we don't have to specify anything actually since it defaults to github.token. Initially I thought we'd have to use our own token, but we can run the checkout action with github.token